docs/tips.md

```diff
@@ -598,20 +598,34 @@ else
 fi
 ```
 
-## Email selected files using Thunderbird
+## Email selected files
 
-To send selected files using Thunderbird, with a keybinding <kbd>Ctrl</kbd> + <kbd>e</kbd>:
+To send selected files using Thunderbird, with a keybinding <kbd>Ctrl</kbd> + <kbd>m</kbd>:
 
 ```toml
 # ~/.config/yazi/keymap.toml
 [[manager.prepend_keymap]]
-on  = "<C-e>"
+on  = "<C-m>"
 run = '''shell --
 	paths=$(for p in "$@"; do echo "$p"; done | paste -s -d,)
 	thunderbird -compose "attachment='$paths'"
 '''
 ```
 
+Or, use the Neomutt terminal email client for sending the selected files with <kbd>Ctrl</kbd> + <kbd>m</kbd>:
+
+```toml
+# ~/.config/yazi/keymap.toml
+[[manager.prepend_keymap]]
+on = "<C-m>"
+run = '''
+	shell '
+        paths=$(for p in "$@"; do echo "$p"; done | tr "\n" " ")
+        ghostty -e neomutt -a $paths
```

Comment:
Why `ghostty -e` here is required?

Reply:
I don't think there's another way to start Neomutt since it's a TUI, similar to Yazi

Reply:
Does `shell --block` not work for you? https://yazi-rs.github.io/docs/configuration/keymap/#manager.shell

Reply:
Aha, didn't know that was possible, i adjusted the code, thanks